Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -11,13 +11,9 @@ If you are new to eye tracking on HoloLens 2 and to how eye tracking is set up i
 We will go into detail on how to address each of them further below.
 
 1. An _'Eye Gaze Data Provider'_ must be added to the input system. This provides eye tracking data from the platform.
-2. The GazeProvider must have its _'Use Eye Tracking'_ property set to true. Note that true is the default value (so no special
-   action is required unless you have actively unchecked this property.)
```

Comment:
Why did you remove this? This is part of the basic instructions to get eye tracking input working in MRTK. It's meant as a short overview check list before going into more detail below. Especially now that by default Eye Tracking is unchecked in most scenes, it's even more important to point this out so that devs can better understand why eye tracking is not working for them.

Reply:
The info is still there. It was moved to a a new topic:

`###` Enabling eye-based gaze in the gaze provider

We are trying to separate instructions for enabling eye tracking from setup eye gaze on gaze provider. This way, user can query eye tracking data without opting for eye based gaze.